systemtest/src/main/java/io/strimzi/systemtest/MessagingBaseST.java

```diff
@@ -219,8 +219,8 @@ private int getReceivedMessagesCount(String response) {
         Pattern r = Pattern.compile(pattern);
```

Comment:
Perhaps you didn't write them, but the above two lines are not readable. The `String.format` does nothing and you might as well inline the `pattern` variable. Please could you fix this?

Reply:
Fixed